src/ts_catalog/continuous_agg.c

```diff
@@ -77,6 +90,51 @@ ts_continuous_agg_with_clause_parse(const List *defelems)
 								 continuous_aggregate_with_clause_def,
 								 TS_ARRAY_LEN(continuous_aggregate_with_clause_def));
 }
+
+List *
+ts_continuous_agg_unparse_compression_defelems(const WithClauseResult *with_clauses)
```

Comment:
We call it "deparse", but I am not sure if this is really deparsing. It looks more like extracting elements from a structure. More like "get names...."
```suggestion
ts_continuous_agg_deparse_compression_defelems(const WithClauseResult *with_clauses)
```

Reply:
I've renamed the method to `ts_continuous_agg_get_compression_defelems`